Alignment/OfflineValidation/plugins/PrimaryVertexValidation.cc

```diff
@@ -266,6 +267,13 @@ void PrimaryVertexValidation::analyze(const edm::Event& iEvent, const edm::Event
 
   edm::ESHandle<GlobalTrackingGeometry> theTrackingGeometry = iSetup.getHandle(trackingGeomToken_);
 
+  //=======================================================
+  // Retrieve tracker topology from geometry
+  //=======================================================
+
+  edm::ESHandle<TrackerTopology> tTopoHandle = iSetup.getHandle(topoToken_);
+  const TrackerTopology* const tTopo = tTopoHandle.product();
```

Comment:
since the `tTopoHandle` is only used to get the product of it, I think the `getData` method could work too
```suggestion
  const TrackerTopology* const tTopo = iSetup.getData(topoToken_);
```

Reply:
this actually won't compile. 
The actual syntax should be `const TrackerTopology* const tTopo = &iSetup.getData(topoToken_);`

Reply:
Ah yes, sorry about missing the `&`